Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -21,13 +22,17 @@ import { vec4, mat4 } from 'gl-matrix';
 const VTK_INT_MAX = 2147483647;
 const { vtkErrorMacro } = macro;
 const viewUpFromViewType = {};
+const defaultViewUpFromViewType = {};
 
 // ----------------------------------------------------------------------------
 // Factory
 // ----------------------------------------------------------------------------
 
 function vtkResliceCursorWidget(publicAPI, model) {
   model.classHierarchy.push('vtkResliceCursorWidget');
+  defaultViewUpFromViewType[ViewTypes.YZ_PLANE] = [0, 0, 1]; // Sagittal
+  defaultViewUpFromViewType[ViewTypes.XZ_PLANE] = [0, 0, 1]; // Coronal
+  defaultViewUpFromViewType[ViewTypes.XY_PLANE] = [0, 1, 0]; // Axial
```

Comment:
can't this go in Constants ?

Reply:
It depends if, for any reason, we want to compute it.

Reply:
when we will want to compute it, then we will get it outside of Constants

Reply:
Sounds good.
Do you know the best way to declare a map with an enum ? Because I tried this but it's not correct :
`const defaultViewUpFromViewType = {
ViewTypes.YZ_PLANE: [0, 0, 1],
ViewTypes.XZ_PLANE: [0, 0, 1],
ViewTypes.XY_PLANE: [0, 1, 0]
};`

Reply:
```
const defaultViewUpFromViewType = { [ViewTypes.YZ_PLANE]: [0, 0, 1], [ViewTypes.XZ_PLANE]: [0, 0, 1], [ViewTypes.XY_PLANE]: [0, 1, 0] };
```